doc/develop/test/twister.rst

```diff
@@ -408,15 +408,22 @@ harness: <string>
     keyboard harness is set on tests that require keyboard interaction to reach
     verdict on whether a test has passed or failed, however, Twister lack this
     harness implementation at the momemnt.
-    The console harness tells Twister to parse a test's text output for a regex
-    defined in the test's YAML file.
 
     Supported harnesses:
 
     - ztest
     - test
     - console
     - pytest
+    - gtest: Use this harness if you've already got tests written in the gTest
```

Comment:
you forgot to remove the "Use this harness.. " from here

Reply:
Oy, sorry, give me 5 minutes.